keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -126,16 +126,16 @@ checklist items _must_ be updated for the enhancement to be released.
 
 Items marked with (R) are required *prior to targeting to a milestone / release*.
 
-- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
-- [ ] (R) KEP approvers have approved the KEP status as `implementable`
-- [ ] (R) Design details are appropriately documented
-- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
-- [ ] (R) Graduation criteria is in place
-- [ ] (R) Production readiness review completed
+- [X] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [X] (R) KEP approvers have approved the KEP status as `implementable`
+- [X] (R) Design details are appropriately documented
+- [X] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [X] (R) Graduation criteria is in place
+- [X] (R) Production readiness review completed
 - [ ] Production readiness review approved
```

Comment:
Did you ever get a PRR reviewer for this?

Reply:
No. I was confused about the the review completed and approved. So I checked it because I completed the questionnaire for readiness review.

Reply:
It looks like it still isn't strictly required: https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#status, but it would be good to get someone to take a look.

cc @prod-readiness-approvers

Reply:
Thanks, let me ask in the prod-readiness channel.

Reply:
That cc didn't appear to work.

cc @wojtek-t

Reply:
i think it needs slash?
/cc @wojtek-t

Reply:
Sorry for large delay - looking into it now.

[And starting 1.20 we will require prod readiness to be approved by PRR team.]

Reply:
@wojtek-t Should I update this PR with an X in  "Production readiness review approved"? or is this typically done by you guys after this merges?

Reply:
Yes - feel free to mark it.